src/Tools/ExternalAccess/Razor/RazorProjectExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    internal static class RazorProjectExtensions
```

Comment:
Don't we keep track of the internal APIs for those EA assemblies as well? If so, I don't see the corresponding change in `InternalAPI.Unshipped.txt`

Reply:
Good catch, addded.